File: projectBackend/projectBackend/Models/LocationTransfers.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace projectBackend.Models
{
  public class LocationTransfers
  {
    [Key]
    public int ID { get; set; }
    [ForeignKey("Flight")]
    public int FlightID { get; set; }
    public virtual Flight Flight { get; set; }
    [ForeignKey("Location")]
    public int LocationID { get; set; }
    public virtual Location Location { get; set; }
    public int Status { get; set; }
  }
}

File: src/app/entities/flights/location-transfers.ts
import { Flight } from './flight'
import { Location } from './location'
import { RideCompany } from '../rides/ride-company'

export class LocationTransfers {
    id:number
    flightID:number
    flight:Flight
    locationID:number
    location:Location
    status: number

    constructor(){
        
    }    
}

export class RideLocationTransfers {
    id:number
    rideCompanyID:number
    rideCompany:RideCompany
    locationID:number
    location:Location
    status: number

    constructor(){
        
    }    
}
